2.4/Dockerfile

```diff
@@ -11,7 +18,20 @@ WORKDIR $HTTPD_PREFIX
 
 # install httpd runtime dependencies
 # https://httpd.apache.org/docs/2.4/install.html#requirements
-RUN set -eux; \
+#
+# When the secret "enable-repro-sources-list" is supplied,
+# /etc/apt is reconfigured to use `snapshot[-cloudflare].debian.org/archive/debian/<SOURCE_DATE_EPOCH>` to
+# reproduce packages. Disabled by default due to the slowness.
```

Comment:
(Note: while `snapshot[-cloudflare].debian.org` is quite slow, `snapshot.ubuntu.com` for Ubuntu seems fast enough to adopt as the default)

https://ubuntu.com/blog/ubuntu-snapshots-on-azure-ensuring-predictability-and-consistency-in-cloud-deployments

Reply:
If this PR is still not acceptable, I'd like to open a PR to add the Ubuntu variant with the fast `snapshot.ubuntu.com`.